langlib/lang.array/src/main/ballerina/src/lang.array/array.bal

```diff
@@ -14,15 +14,25 @@
 // specific language governing permissions and limitations
 // under the License.
 
+# A type parameter that is a subtype of `any|error`.
+# Has the special semantic that when used in a declaration
```

Comment:
```suggestion
# Use the special semantic when used in a declaration.
```
The special sematic of what? Maybe, explain with "i.e.,".

Reply:
This description about type parameter is directly taken from the spec. Shall we leave it as is.